src/components/Footer/Footer.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+
+const Footer = () => {
+  const location = useLocation();
+
+  let isCheckPath = location.pathname === '/';
+
+  return (
+    <FooterContainer>
+      <Inner
+        style={{
+          maxWidth: isCheckPath ? 'none' : '1120px',
+          padding: isCheckPath ? '0 80px' : '0',
+        }}
+      >
+        <FooterTop>
+          <span>© 2022 AirWnB Inc.</span>
+          <FooterUl>
+            <li>
+              <i class="fa-brands fa-facebook" />
+            </li>
+            <li>
+              <i class="fa-brands fa-instagram" />
+            </li>
+            <li>
+              <i class="fa-brands fa-twitter" />
+            </li>
+            <li>
+              <i class="fa-brands fa-youtube" />
+            </li>
+          </FooterUl>
+        </FooterTop>
+        <FooterBottom>
+          <span>
+            웹사이트 제공자: Airbnb Ireland UC, private unlimited company, 8
+            Hanover Quay Dublin 2, D02 DP23 Ireland | 이사: Dermot Clarke,
+            Killian Pattwell, Andrea Finnegan | VAT 번호: IE9827384L | 사업자
+            등록 번호: IE 511825 | 연락처: terms@airbnb.com, 웹사이트,
+            080-822-0230 | 호스팅 서비스 제공업체: 아마존 웹서비스 |
+            에어비앤비는 통신판매 중개자로 에어비앤비 플랫폼을 통하여 게스트와
+            호스트 사이에 이루어지는 통신판매의 당사자가 아닙니다. 에어비앤비
+            플랫폼을 통하여 예약된 숙소, 체험, 호스트 서비스에 관한 의무와
+            책임은 해당 서비스를 제공하는 호스트에게 있습니다.
+          </span>
+        </FooterBottom>
+      </Inner>
+    </FooterContainer>
+  );
+};
+
+const FooterContainer = styled.div`
+  border: 1px solid #eee;
+  padding: 30px 0;
+`;
+
+const Inner = styled.div`
+  width: 100%;
+  max-width: 1120px;
+  margin: 0 auto;
+`;
```

Comment:
```suggestion
const Inner = styled.div`
  width: 100%;
  max-width: ${props => (props.path ? 'none' : '1120px')};
  padding: ${props => (props.path  ? '0 80px' : '0')};
  margin: 0 auto;
`;
```

이런식으로 컴포넌트에서 넘겨준 props로 스타일을 동적으로 관리할 수 있습니다.